src/Moontoast/Math/BigNumber.php

```diff
@@ -647,10 +647,11 @@ public static function convertFromBase10($number, $toBase)
         $outNumber = '';
         
         while (bccomp($number, $toBase) >= 0) {
-            $quotient = bcdiv($number, $toBase, 0);
-            $remainder = bcsub($number, bcmul($quotient, $toBase, 0), 0);
+            list($number, $remainder) = array(
+                bcdiv($number, $toBase, 0),
+                bcmod($number, $toBase)
+            );
```

Comment:
I understand why using `bcmod()` is more expressive, but I don't understand why wrapping them in an `array()` and using `list` is more expressive. Couldn't this be more easily understood as:'

``` php
$number = bcdiv($number, $toBase, 0);
$remainder = bcmod($number, $toBase);
```